brokerapi/brokers/bigquery/broker.go

```diff
@@ -59,7 +60,7 @@ func (b *BigQueryBroker) Provision(instanceId string, details models.ProvisionDe
 		params["name"] = name_generator.Basic.InstanceName()
 	}
 
-	service, err := googlebigquery.New(b.Client)
+	service, err := googlebigquery.New(b.HttpConfig.Client(context.Background()))
```

Comment:
the user agent is either set on the service (as here, see line 67) or in ClientOptions, depending on which format the constructor uses.